src/lib/Components/Bidding/Components/BackButton.tsx

```diff
@@ -0,0 +1,29 @@
+import React from "react"
+import { NavigatorIOS, TouchableWithoutFeedback, ViewProperties } from "react-native"
+
+import { Image } from "../Elements/Image"
+import { theme } from "../Elements/Theme"
+
+interface ContainerWithBackButtonProps extends ViewProperties {
+  navigator: NavigatorIOS
+}
+
+export class BackButton extends React.Component<ContainerWithBackButtonProps> {
```

Comment:
I'm not sure what the convection is in Emission, but this seems like a thing that could live in a more global space eventually.